### PR TITLE
Fix gym envs viewer windows not closing

### PR DESCRIPTION
**Pull request by gitanshu**
[The nightly tests are passing now on CI](https://travis-ci.com/rlworkgroup/garage/builds/115189123)

### CODE REVIEW

File: tests/helpers.py [hunk 1]
```diff
@@ -33,24 +33,28 @@ def step_env_with_gym_quirks(test_case,
...
+        _, _, done, _ = _env.step(_env.action_space.sample())
         if render:
             if spec.id not in KNOWN_GYM_RENDER_NOT_IMPLEMENTED:
-                env.render()
+                _env.render()
             else:
                 with test_case.assertRaises(NotImplementedError):
-                    env.render()
+                    _env.render()
+            if hasattr(env.env, 'viewer') and env.env.viewer is None:
```
Comment (ryanjulian) [inline tests/helpers.py]: this is really hacky and hard to understand. it also makes assumptions about the caller.

is there a reason you can't just close the viewer on the serialized environment here, immediately?

Comment (gitanshu) [reply tests/helpers.py]: I agree. And yeah, closing window on serialized environment makes more sense. I'll update it.

**Comment by ryanjulian** [comment]
LGTM but please bring the test coverage up.

**Comment by ryanjulian** [comment]
Again please prioritize this. Our builds go red every night :/

**Comment by gitanshu** [comment]
Can one of you merge this then?

**Comment by ryanjulian** [comment]
Please update the branch and we can force-push it for you.

**Pull request merged**